images/linux/scripts/installers/aliyun-cli.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+################################################################################
+##  File:  aliyun-cli.sh
+##  Desc:  Installs Alibaba Cloud CLI
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install Alibaba Cloud CLI
+URL=$(curl -s https://github.com/aliyun/aliyun-cli | grep -m1 ">Linux<" | cut -d "\"" -f 2)
```

Comment:
is this resilient enough? would it make sense to go to the Releases instead to download from there?